### DISCUSSION TITLE
ASIDE: tabular json and data packages ...

### DISCUSSION THREAD

**Discussion by rufuspollock** [frictionlessdata/frictionlessdata.io#649] [General]
It should be natural to use Table Schema and "tabular" json in data packages (it is kind of what we do when we cache CSV data onto a resource as "_values").

It would be nice to have an example of this.

**Comment by lwinfree** [comment]
@roll is there an example of this now with Frictionless-py?

**Reply by roll** [reply]
Not sure exactly. Do you mean Inline Data - https://framework.frictionlessdata.io/docs/tutorials/formats/inline-tutorial?

If yes, we need to add some example of how it can be altered using the Schema
